jupyterlab/labapp.py

```diff
@@ -729,6 +730,8 @@ def initialize_handlers(self):  # noqa
 
         # Set config for Jupyterlab
         page_config = self.serverapp.web_app.settings.setdefault("page_config_data", {})
+        page_config.update(get_static_page_config(logger=self.log, level="sys_prefix"))
```

Comment:
I believe this should also show plugins disabled at `user` and `system` level, right? 

```suggestion
        page_config.update(get_static_page_config(logger=self.log, level="all"))
```